cirq-google/cirq_google/api/v2/run_context.proto

```diff
@@ -100,6 +106,38 @@ message DeviceParameter {
     // by the sweep values themselves.
 }
 
+// A bundle of multiple DeviceParameters and their values.
+// The main use case is to set those parameters with the
+// values from this bundle before executing a circuit sweep.
+// See run_context.py for utility function to construct a
+// DeviceParametersDiff from a list of (DeviceParameter, ArgValue) pairs.
+message DeviceParametersDiff {
```

Comment:
what does a repeated `attributes` field mean?

Reply:
I think this is just a typo. A map already incorporates multiple map entries.

Reply:
instead of inventing a new message as you allude to, the proposed DeviceParametersDiff is based on internal RegistryDiff data structure, which is well tested and in production uses.
I would prefer to stay with the current DeviceParametersDiff definition. wdyt?